inc/functions.php

```diff
@@ -2299,11 +2275,10 @@ function strip_combining_chars($str) {
 function buildThread($id, $return = false, $mod = false) {
 	global $board, $config, $build_pages;
 	$id = round($id);
-    print_err("buildThread start");
 
-	if (event('build-thread', $id)) {
-		return;
-    }
+  if (event('build-thread', $id)) {
```

Comment:
this was just formatting

Reply:
The indent looks inconsistent from here.

Reply:
Yes it was a tabs vs spaces issue. I'm probably going to start converting things to 4 spaces. This has been fixed on myownbranch but I only saw your comment after I already merged.